tests/drivers/flash/erase_blocks/src/main.c

```diff
@@ -38,12 +38,12 @@ LOG_MODULE_REGISTER(test_flash);
 	DT_MTD_FROM_FIXED_PARTITION(TEST_FLASH_PART_NODE)
 
 static const struct device *flash_controller = DEVICE_DT_GET(TEST_FLASH_CONTROLLER_NODE);
-static uint8_t test_write_block[128];
-static uint8_t test_read_block[128];
+static uint8_t test_write_block[512];
+static uint8_t test_read_block[512];
 
 static void test_flash_fill_test_write_block(void)
 {
-	for (uint8_t i = 0; i < sizeof(test_write_block); i++) {
+	for (size_t i = 0; i < sizeof(test_write_block); i++) {
 		test_write_block[i] = i;
```

Comment:
CI may complain if you don't explicitly cast `i` to `(uint8_t)`

Reply:
> CI may complain if you don't explicitly cast `i` to `(uint8_t)`

This is the intentional fix. 
uint8_t may not be used here as it is overflowed by sizeof() returning unsigned int.
So uint8_t must be replaced by size_t (unsigned int).

Reply:
> CI may complain if you don't explicitly cast `i` to `(uint8_t)`

The typecast has been added. Thanks.

Reply:
Yeah, I made the same mistake :) Github shows more lines than the 1 line (line 47) I actually selected...